### ISSUE TITLE
[FlexibleColumnLayout]: horizontal splitter is read for vertical splitter

### ISSUE THREAD

**Issue by judith9209** [UI5/webcomponents#13702]
### Bug Description

open https://ui5.github.io/webcomponents/nightly/components/fiori/FlexibleColumnLayout/ and set focus on the splitter: Screen reader reads "horizontal splitter":  
<img width="598" height="404" alt="Image" src="https://github.com/user-attachments/assets/65754e4d-9a56-41ef-8b78-9ee27812faf4" />

### Affected Component

FlexibleColumnLayout

### Expected Behaviour

Screen reader should read vertical splitter like in UI5. https://ui5.sap.com/#/entity/sap.f.FlexibleColumnLayout/sample/sap.f.sample.FlexibleColumnLayoutWithFullscreenPage 
<img width="809" height="544" alt="Image" src="https://github.com/user-attachments/assets/41dc2b5c-67e3-4256-903c-d7426dc57944" />

aria-orientation="vertical" might be missing.

### Steps to Reproduce

1. Open https://ui5.github.io/webcomponents/nightly/components/fiori/FlexibleColumnLayout/ 
2. Switch on a scree reader
3. Set focus on the splitter
...

### Priority

None

### UI5 Web Components Version

nightly

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by plamenivanov91** [comment]
Hello @UI5/ui5-team-ctr-pir ,

I can confirm that the splitter orientation is mistaken.

Could you please fix it?

Regards,
Plamen Ivanov
